org.eclipse.cdt.cross.arc.gnu.uclibc/plugin.xml

```diff
@@ -3849,20 +3807,9 @@
             errorParsers="org.eclipse.cdt.core.GCCErrorParser"
             id="org.eclipse.cdt.cross.arc.gnu.eb.uclibc.cpp.compiler"
             isAbstract="false"
-            name="ARC uClibc GCC C++ Compiler (eb)"
+            name="ARC uClibc GCC C++ (eb)Compiler"
```

Comment:
I'm not sure new ordering is right here.. Plus lets use "(eb)" to "(EB)" across the files.